<launch>  
  <arg name="ip" default="FLOWERS-DELL-1.local"/> <!-- IP of the VRPN server (like Motive or Arena) -->
  <arg name="port" default="3883" /> <!-- Port of the VRPN server (default 3883) -->
  <arg name="num_skeletons" default="1" /> <!-- Number of skeletons to track -->

  <param name="kinect/vrpn_ip" type="string" value="$(arg ip)" />
  <param name="kinect/vrpn_port" type="string" value="$(arg port)" />
  <param name="kinect/num_skeletons" type="int" value="$(arg num_skeletons)" />
  <!-- <param name="kinect/use_v2" type="bool" value="$(arg use_v2)" /> -->

  <node name="kinect_skeleton_publisher" pkg="kinect_skeleton_publisher" type="skeleton_publisher.py" output="screen"/>

</launch>
